e2e/testconfig/testconfig.go

```diff
@@ -370,25 +382,30 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
 				Version:    cc.Tag,
 			},
 		},
-		Bin:                 cc.Binary,
-		Bech32Prefix:        "cosmos",
-		CoinType:            fmt.Sprint(sdk.GetConfig().GetCoinType()),
-		Denom:               denom,
-		GasPrices:           fmt.Sprintf("0.00%s", denom),
-		GasAdjustment:       1.3,
-		TrustingPeriod:      "508h",
-		NoHostMount:         false,
-		ModifyGenesis:       getGenesisModificationFunction(cc),
-		ConfigFileOverrides: configFileOverrides,
+		Bin:                    cc.Binary,
+		Bech32Prefix:           "cosmos",
+		CoinType:               fmt.Sprint(sdk.GetConfig().GetCoinType()),
+		Denom:                  denom,
+		GasPrices:              fmt.Sprintf("0.00%s", denom),
+		GasAdjustment:          1.3,
+		TrustingPeriod:         "508h",
+		NoHostMount:            false,
+		ModifyGenesis:          getGenesisModificationFunction(cc),
+		ConfigFileOverrides:    configFileOverrides,
+		UsingNewGenesisCommand: useNewGenesisCommand,
```

Comment:
This is a field specified by interchain test? Seems a bit odd in my opinion since this seems to be very cosmos specific?

Reply:
yes it is, my understanding of how the cobra fields are defined is that any chain could specify whatever args they want, which means the command interchain test ends up using might not actually be correct. It is however in this case. I think there is probably a more generic solution that could be used somewhere though.